Firestore/core/src/firebase/firestore/local/leveldb_key.h

```diff
@@ -142,7 +143,7 @@ class LevelDbMutationKey {
  private:
   // Deliberately uninitialized: will be assigned in Decode
   std::string user_id_;
-  model::BatchId batch_id_;
+  model::BatchId batch_id_ = model::kBatchIdUnknown;
```

Comment:
What's the reason for the changes in this file?

Reply:
I caught an uninitialized field elsewhere, searched for other possibly related fields, found these and and decided that the deliberate uninitialized fields in here were just a liability. Apologies for failing to clean up the comments.